src/to-have-prop.ts

```diff
@@ -1,17 +1,23 @@
 import { matcherHint, stringify, printExpected } from 'jest-matcher-utils';
+import { ReactTestInstance } from 'react-test-renderer';
 import { checkReactElement, getMessage } from './utils';
 
-function printAttribute(name, value) {
+function printAttribute(name: string, value: any) {
```

Comment:
```suggestion
function printAttribute(name: string, value: unknown) {
```

Reply:
Fixed, added eslint rule for the future that checks explicit any.